README.md

```diff
@@ -57,6 +57,10 @@ mavenCentral {
     // Token for Publisher API calls obtained from Sonatype official,
     // it should be Base64 encoded of "username:password".
     authToken = '<your token>'
+
+    // Whether the upload should be automatically published or not.
+    // Use `USER_MANAGED` if you wish to do this manually.
+    publishingType.set(PublishingType.AUTOMATIC)
```

Comment:
It can't be set with quotes (as it's not a String type) but I was able to remove the `set` method call. This will work: `publishingType = PublishingType.AUTOMATIC`. Is that okay?

I have added the comment.